pkg/kubectl/cmd/config/current_context_test.go

```diff
@@ -66,6 +66,7 @@ func (test currentContextTest) run(t *testing.T) {
 
 	pathOptions := NewDefaultPathOptions()
 	pathOptions.GlobalFile = fakeKubeFile.Name()
+	pathOptions.EnvVar = ""
```

Comment:
would it be better to clear the envvar itself and defer restoring the original value?